staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1beta1/register.go

```diff
@@ -44,6 +47,9 @@ func init() {
 
 // Adds the list of known types to the given scheme.
 func addKnownTypes(scheme *runtime.Scheme) error {
+	scheme.AddKnownTypes(LegacySchemeGroupVersion,
+		&EgressSelectorConfiguration{},
```

Comment:
why add a new type to the legacy group?

Reply:
Oh, I missed the constant value changed. Changing constant values is a bit squirrelly… would like to avoid silently breaking things referring to those constants

Reply:
Updated to introduce a new constant instead of renaming.